controlplane/kubeadm/controllers/suite_test.go

```diff
@@ -28,6 +30,25 @@ import (
 var (
 	env *envtest.Environment
 	ctx = ctrl.SetupSignalHandler()
+	// TODO: move under internal/testtypes (or refactor it in a way that we don't need it anymore).
```

Comment:
```suggestion
	// TODO @sbueringer: move under internal/testtypes (or refactor it in a way that we don't need it anymore).
```

Best to say who is adding the TODO for future git archaeologists.

Reply:
Agree. Minor diff: I used the `TODO(github-id)` pattern, as that's the one which seems to be used in k/k